core/relayMsgs.go

```diff
@@ -161,6 +185,14 @@ func (r *RelayMsgs) Send(src, dst Chain) {
 	r.DstMsgIDs = dstMsgIDs
 }
 
+func msgsToLoggable(msgs []sdk.Msg) []string {
+	var ret []string
+	for _, msg := range msgs {
+		ret = append(ret, fmt.Sprintf("%T{%v}", msg, msg))
```

Comment:
nit: Can't we use `%#v` instead of `%T{%v}`?

Reply:
Thank you, I forgot this for a long time. I will fix it.

Reply:
fixed: https://github.com/hyperledger-labs/yui-relayer/pull/143/commits/c086416db7a5e0f2b07089dd4b10ed06efa06e64